mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/domain/JobDefinition.java

```diff
@@ -133,12 +133,13 @@ public void validate(boolean schedulingInfoOptional) throws InvalidJobException
     }
 
     public boolean requireInheritInstanceCheck() {
-        return this.schedulingInfo != null && this.deploymentStrategy != null && this.getDeploymentStrategy().requireInheritInstanceCheck();
+        return this.schedulingInfo != null &&
```

Comment:
I just want to confirm that my understanding is correct. Are we considering the absence of a deploymentStrategy as indicating that we need to have the same worker count as the previous job?

Reply:
yes